external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/v2/V2JDBCTest.scala

```diff
@@ -348,4 +354,145 @@ private[v2] trait V2JDBCTest extends SharedSparkSession with DockerIntegrationFu
       }
     }
   }
+
+  protected def checkFiltersRemoved(df: DataFrame): Unit = {
```

Comment:
how is this different from `filterPushed`?

Reply:
I didn't see it